docs/current_docs/api/playground.mdx

```diff
@@ -0,0 +1,11 @@
+---
+slug: /api/playground
```

Comment:
This is a standalone page which does not appear in the docs sidebar. Requests to `https://play.dagger.cloud/playground/*` should redirect here. Redirection may be tested from the docs archive page at [https://archive.docs.dagger.io/0.9/api/282105/playground](https://archive.docs.dagger.io/0.9/api/282105/playground)